demo/vl-modal.html

```diff
@@ -209,5 +209,22 @@ <h2>Safari shadow DOM focus</h2>
+                    &lt;vl-modal&gt;&lt;/vl-modal&gt;
```

Comment:
Komt niet overeen met de code. Binnenkort (wanneer de vl-ui-demo webcomponent er is) is dit manueel werk gelukkig niet meer nodig.